etc/templates/profile.template

```diff
@@ -176,6 +176,7 @@ include globals.local
 #noroot
 #nosound
 #notv
+#notpm
```

Comment:
Firejail likes alphabetically ordering options (in profiles). Move it accordingly.